lib/neutron

```diff
@@ -359,7 +359,8 @@ function install_neutron_new {
 
     # L3 service requires radvd
     if is_service_enabled neutron-l3; then
```

Comment:
maybe just disable the l3 service?  Is the issue here that radvd isn't present in clear?

Reply:
radvd should have been included in the openstack-common.